### ISSUE TITLE
The README's installation part is confusing

### ISSUE THREAD

**Issue by kuchaguangjie** [abice/go-enum#251]
Why not simply say:

> go install github.com/abice/go-enum@latest

**Comment by abice** [comment]
@kuchaguangjie Yeah, I can absolutely add that as part of the readme as another option, but the released binaries have build information embedded in them, which is preferable for a lot of folks.  go install does not provide a way for me to build the release version into the binary that you run locally.

**Comment by abice** [comment]
I've updated the readme with latest updates, and includes a quickstart which just has the go install instruction
